docs/cost_estimates.md

```diff
@@ -0,0 +1,98 @@
+# Costs and financing
+
+In this section, we will discuss some of the financial aspects which are important to as you prepare an manage an open source project.
+
+## A first cost overview
+
+In general we can divide the cost into four major categories:
+
+- **Staff**: The staff includes the people working on the project like developers, testers, Q&A and others, their travel etc.
+- **Hardware**: The hardware that is needed to get the whole project also running.
+- **Administration**: Who is working behind the scences to get the non-technical questions resolved ?
+- **Licences**: It is fairly likely that you will have some licence costs (working on open-source or not).
```

Comment:
Maybe there's the relevant maintenance or service level agreements missing for product or 3rd level support? This would be both for open source and non open source